### ISSUE TITLE
RB Mixer not working

### ISSUE THREAD

**Issue by noamkrief** [FrSkyRC/RF_Feedback_Group#51]
![image](https://github.com/user-attachments/assets/3633f2f3-9693-44c6-bd30-c28fa067ba75)

This is what happens when I try to "WRITE" to the RB35S.

I don't know who wrote this software, but it seems VERY powerful.
The documentation is not very good, so I'll have to make many experiments to see how everything works.

But for now, I cannot write this to the RB35S using the STK tool.

Can someone help me out?
Thanks

**Comment by KAID-YANG** [comment]
1. First, set the IMU mode option in the RB35S LUA script to ADV mode.  
2. Connect the RX1 S.P interface of the RB35S to the STK (no need to connect the positive terminal).  
3. Power on the RB35S and then connect the STK tool to the computer.  
4. Open the RB MIXER tool.  
5. Select the COM port.  
6. Choose Read.  
7. Edit the parameters for self-stabilization in ADV mode.  
8. Select Write.  
9. Editing is complete.

**Comment by KAID-YANG** [comment]
https://drive.google.com/file/d/16gM8J9iv-jo7k2M1RGn2tyOCHyajDXJC/view?usp=drive_link

**Comment by noamkrief** [comment]
@KAID-YANG 
It works now! My problem was "(no need to connect the positive terminal)."

Can you upload a db file sample, so I can see how you program each line?
Thank you so much!!!!

**Comment by KAID-YANG** [comment]
This is a delta wing example file, with output on CH17 & CH18. The settings are for reference only and not recommended for direct copying.
[Delta2.zip](https://github.com/user-attachments/files/17357974/Delta2.zip)

**Comment by doppleganger53** [comment]
> 1. First, set the IMU mode option in the RB35S LUA script to ADV mode.
> 2. Connect the RX1 S.P interface of the RB35S to the STK (no need to connect the positive terminal).
> 3. Power on the RB35S and then connect the STK tool to the computer.
> 4. Open the RB MIXER tool.
> 5. Select the COM port.
> 6. Choose Read.
> 7. Edit the parameters for self-stabilization in ADV mode.
> 8. Select Write.
> 9. Editing is complete.

Should the same steps work with a TD SR12 with IMU in ADV mode & config enable? Following these steps, it seems that the RBMixer tool just loops while reading the parameters, as if it were reading telemetry and not the config parameters.

**Comment by KAID-YANG** [comment]
> > 1. First, set the IMU mode option in the RB35S LUA script to ADV mode.
> > 2. Connect the RX1 S.P interface of the RB35S to the STK (no need to connect the positive terminal).
> > 3. Power on the RB35S and then connect the STK tool to the computer.
> > 4. Open the RB MIXER tool.
> > 5. Select the COM port.
> > 6. Choose Read.
> > 7. Edit the parameters for self-stabilization in ADV mode.
> > 8. Select Write.
> > 9. Editing is complete.
> 
> Should the same steps work with a TD SR12 with IMU in ADV mode & config enable? Following these steps, it seems that the RBMixer tool just loops while reading the parameters, as if it were reading telemetry and not the config parameters.

Configuration steps for TD SR12:
1. First, set the IMU mode to ADV mode in the TDSR12 LUA script.
2. Set the ADV config option to Enable.
3. Connect the STK tool to the Upgrade interface of the TDSR12.
4. Connect the STK tool to the computer, then open RB MIXER.
5. Select the COM port.
6. Choose "Read."
7. Edit the parameters for self-stabilization in ADV mode.
8. Select "Write."
9. Editing is complete.
10. After the setup is complete, set the ADV config to Disable.

**Comment by noamkrief** [comment]
@KAID-YANG 
Hello, please help us.
I used your delta2.db and successfully wrote to RB35S

I hooked up a servo to Channel 17 on the RB35S but as I move the RB35S around, the servo on channel 17 does not move.

Should the server at Channel 17 move when I move the RB35S in my hands?

**Comment by Dsrenger** [comment]
Did you bind the receiver with 1-24ch?

Everytime channel 17 and 18 do not move for me, I falsely bound the receiver with 1-16 channels

**Comment by noamkrief** [comment]
@Dsrenger yes - I'm at 1-24ch mode. Have you ever gotten this rbmixer to work?
@doppleganger53 Have you got this to work?

**Comment by KAID-YANG** [comment]
> @KAID-YANG Hello, please help us. I used your delta2.db and successfully wrote to RB35S
> 
> I hooked up a servo to Channel 17 on the RB35S but as I move the RB35S around, the servo on channel 17 does not move.
> 
> Should the servo at Channel 17 move when I move the RB35S in my hands?
> 
> I'm in ADV mode.

"Please provide the document you edited. For each channel's stabilization, you also need to change the `enable` value from 0 to 1 in the lower part of the RBmixer interface, which indicates that stabilization for that channel is enabled."

**Comment by noamkrief** [comment]
@KAID-YANG 
I used the file you sent delta2.db. I didn't modify it. Just copied it to the RB35S
